### PR TITLE
Add types for airbrake-js@1.0.x

### PR DESCRIPTION
**Pull request by stripedpajamas**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## airbrake-js ([<kbd>unpkg</kbd>](https://unpkg.com/browse/airbrake-js@latest/))
was missing the following properties:
1. detectReporter
1. errors
1. truncate
1. Promise
1. historian

<details>
<summary>as well as these 3 other properties...</summary>
<p>getHistory, makeEventHandler, Client</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 0fdc6e4a8ae0f49f2a67355b86d873239964539a
</p>

**Comment by martin-badin** [comment]
Are you sure you want to add types definition for no longer maintained package?

File: types/airbrake-js/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
+// Type definitions for airbrake-js 1.0
+// Project: https://github.com/airbrake/airbrake-js/tree/c4516f3848099b9d1b8102a5584e6b38c5b0d4c2
```
Comment (martin-badin) [inline types/airbrake-js/index.d.ts:2]: update URL to https://github.com/airbrake/airbrake-js

**Review by martin-badin — CHANGES_REQUESTED** [review]

**Comment by stripedpajamas** [comment]
> Are you sure you want to add types definition for no longer maintained package?

Actually... wtf am I doing.

**Pull request closed without merging**